app/src/lib/git/merge.ts

```diff
@@ -9,15 +13,73 @@ export async function merge(
   await git(['merge', branch], repository.path, 'merge')
 }
 
+/**
+ * Find the base commit between two refs
+ *
+ * @returns the commit id of the merge base, or null if the two refs do not
```

Comment:
We should document what type of string we're looking for for the _*Ref_ parameters (like is it expecting a name of a branch or tag, a commit, or some other value).

Reply:
The docs for `merge-base` say `<commit>` but my testing indicates this can be any commit/ref/tag/`HEAD`. I'll make it clearer in the title (documenting each `@param` felt noisy at the time):

> Find the base commit between two refs

Reply:
Yea, changing the name of the param to `committish` may also work. I agree that having the param docs is a bit noisey.a